ReportSimulationOutput/measure.rb

```diff
@@ -870,6 +871,25 @@ def get_outputs(runner, timeseries_frequency,
       @totals[TE::Net].timeseries_output = @totals[TE::Total].timeseries_output.zip(outputs[:elec_prod_timeseries]).map { |x, y| x + y * unit_conv }
     end
 
+    # Resilience Hours
+    @resilience_hours.each do |key, resilience_hour|
+      next unless key == RHT::Battery
+
+      resilience_hour.variables.map { |v| v[0] }.uniq.each do |sys_id|
+        keys = resilience_hour.variables.select { |v| v[0] == sys_id }.map { |v| v[1] }
+        vars = resilience_hour.variables.select { |v| v[0] == sys_id }.map { |v| v[2] }
+
+        batt_kwh = @hpxml.batteries[0].usable_capacity_kwh
+        batt_kw = @hpxml.batteries[0].rated_power_output
+        batt_roundtrip_efficiency = 0.95
+        batt_soc_kwh = get_report_variable_data_timeseries(keys, vars, 1, 0, timeseries_frequency)
+        crit_load = outputs[:elec_net_timeseries]
```

Comment:
@joseph-robertson: Close, but needs a couple of changes:
- The SoC in kWh would be the usable capacity multiplied by the SoC (SoC is 0-1). We also need to account for our min SoC not being 0, so I think we would calculate this as (get_report_variable_data_timeseries(keys, vars, 1, 0, timeseries_frequency)- minimum_storage_state_of_charge_fraction) * batt_kWh
-  Might be a good idea to make a new variable for the battery roundtrip efficiency and use that in the battery model and here, rather than relying on a hard coded value matching the default in case we ever change this. 
- Critical load seems fine. In the future we could choose only certain loads to be "critical", but I think whole home net power (treating every load as critical) is good for the initial implementation.